docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: '2.2'
+
+# data volume for DB
+volumes:
+  dbstorage:
+
+# services (docker containers)
+services:
+  # MariaDB database
+  db:
+    image: mariadb:10.3
+    # Set max_allowed_packet to 256M
+    command: --max_allowed_packet=32505856
+    restart: always
+    container_name: variantstore_db
+    environment:
+      MYSQL_ALLOW_EMPTY_PASSWORD: "yes"
+      MYSQL_DATABASE: variantstore_db
+    volumes:
+     - dbstorage:/data/db
+     - ./models/variantstore-model.sql:/docker-entrypoint-initdb.d/variantstore-init.sql:ro
+     - ./models/transaction-db.sql:/docker-entrypoint-initdb.d/transaction-init.sql:ro
+    ports:
+     - 3306:3306
+    healthcheck:
+      test: "/usr/bin/mysql --user=root --password='' --database=variantstore_db --execute \"SELECT * FROM variant_has_variantcaller;\""
+      interval: 5s
+      timeout: 10s
+      retries: 100
+  # Variantstore service
+  variantstore:
+    container_name: variantstore_app
+    build: .
+    image: variantstore:latest
+    links:
+      - db
+    depends_on:
+      db:
+        condition: service_healthy
+    ports:
+      - 8080:8080
+    environment:
```

Comment:
We would also need to set the other environment variables for the Variantstore here, right? (such as `VARIANTSTORE_OAUTH2_ENABLED` e.g.

I guess it would be better if we switch to `.env` files as described [here](https://docs.docker.com/compose/environment-variables/).

Reply:
Correct, would increase the overview on configs definitely.

Reply:
Added variantstore specific configs with deactivated seetings as default.
Maybe, you can add here some better default configurations in the future for OAUTH2.